.github/workflows/scripts/slack_notifications/slack_notifier.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'httparty'
+
+class SlackNotifier
+  CYCLE = 24 * 60 * 60 # Period in seconds to check for updates that need to be Slacked."
+
+  def self.send_slack_message(message)
+    path = ENV['SLACK_GEM_NOTIFICATIONS_WEBHOOK']
+    options = {headers: {'Content-Type' => 'application/json'},
+               body: {text: message}.to_json}
+
+    HTTParty.post(path, options)
```

Comment:
What happens if this POST request fails? How can we discover the failure?

Reply:
@kaylareopelle I always assumed the CI would post the failure and the action would fail. Do you recommend doing some kind of catch here for failures?

Reply:
I think a begin / rescue block wrapped around this line would be a great idea to catch the failures and possibly print them to the CI log. (reading your specific comment vs. talking about it syncronously sparked a slightly different answer, though I'm still curious to see what happens if the post fails in the test we discussed)

Reply:
This looks a little different from when we worked on it, but let me know if you have questions. [Here's the code](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-28d6e09aae9f0b8a06eaa86323bb371cd4ea0ab2b399e13d0e89ca760b7bf704R10-R31).